projects/slap_manipulation/scripts/rename_trials.py

```diff
@@ -0,0 +1,40 @@
+"""Script to fix names of trials s.t. they are unique
+Only required for the few files collected using prev version of collect_h5.py
+"""
+import glob
+import os
+
+import click
+import h5py
+
+
+@click.command()
+@click.option("--data-dir", default="./", help="Path where the files are stored")
+@click.option(
+    "--template",
+    default="*.h5",
+    help="Files will be looked up using regex: data-dir/template",
+)
+@click.option("--mode", type=click.Choice(["read", "write"]), default="read")
+def main(data_dir, template, mode):
```

Comment:
how do you tell it what to change/rename? does not seem to be an argument

Reply:
This one is defunct. Removed in latest commits.